mozilla-release/mobile/android/base/java/org/mozilla/gecko/controlcenter/OverviewFragment.java

```diff
@@ -105,7 +105,7 @@ public void onConfigurationChanged(Configuration newConfig) {
                 .beginTransaction()
                 .detach(this)
                 .attach(this)
-                .commit();
+                .commitAllowingStateLoss();
```

Comment:
based on documentation: https://developer.android.com/reference/android/app/FragmentTransaction.html#commitAllowingStateLoss()
"This is dangerous because the commit can be lost if the activity needs to later be restored from its state"
is the app works fine when you press HomeButton and return back to the app ?